rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -1,87 +1,41 @@
 :_content-type: REFERENCE
 [id="rosa-release-notes"]
-= {product-title} 4.10 release notes
+= What's new with {product-title}
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-release-notes
 
 toc::[]
 
 {product-title} (ROSA) is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
 
-[id="about-this-release-section_{context}"]
-== About this release
+{product-title} clusters are available at https://console.redhat.com/openshift. With the Red Hat OpenShift Cluster Manager application for ROSA, you can deploy {product-title} clusters to either on-premises or cloud environments.
```

Comment:
The reference to https://console.redhat.com/openshift below has link text. Should this one also?